YTSpammerPurge.py

```diff
@@ -286,7 +284,8 @@ class MiscDataStore:
 
 
   #----------------------------------- Begin Showing Program ---------------------------------
-  print(f"{F.LIGHTYELLOW_EX}\n===================== YOUTUBE SPAMMER PURGE v" + version + f" ====================={S.R}")
+  print(f"{F.LIGHTYELLOW_EX}\n===================== YOUTUBE SPAMMER PURGE v{version} ====================={S.R}")
```

Comment:
```suggestion
  print(f"{F.LIGHTYELLOW_EX}\n===================== YT-SPAMMER-PURGE v{version} ====================={S.R}")
```

Reply:
YT = YouTube

I think it should remain YOUTUBE SPAMMER PURGE without the dashes. Idk, maybe that's me.

Reply:
> I think it should remain YOUTUBE SPAMMER PURGE without the dashes. Idk, maybe that's me.
YT-Spammer-Purge is the repository name, it's best to remain consistent in my opinion 😁